### PR TITLE
Feat/#453 Implement ErrorOutOfGasSSTORE/ErrorOutOfGasSLOAD error state

### LINKED ISSUE #453: Implement `ErrorOutOfGasSSTORE` error state
See https://github.com/privacy-scaling-explorations/zkevm-specs/pull/381 for more details

### PR DESCRIPTION
**Pull request by KimiWu123**
fixed #453 and #452

### CODE REVIEW

File: src/zkevm_specs/evm_circuit/execution/error_oog_sload_sstore.py [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+        instruction.constrain_equal(insufficient_gas, FQ(1))
+    else:
+        # For SSTORE, the OOG error occurs when the gas left is less than or equal to `SSTORE_SENTRY`
+        lt_gas, eq_gas = instruction.compare(
+            instruction.curr.gas_left, FQ(GAS_COST_SSTORE_SENTRY_EIP2200), N_BYTES_GAS
+        )
+        instruction.constrain_not_zero(lt_gas + eq_gas + insufficient_gas)
+
+    instruction.constrain_error_state(
+        instruction.rw_counter_offset + instruction.curr.reversible_write_counter + 1
```
Comment (ChihChengLiang) [inline src/zkevm_specs/evm_circuit/execution/error_oog_sload_sstore.py:59]: Can we doc the reason for `+1`?

**Review by ChihChengLiang — APPROVED** [review]
LGTM with a nitpick.

File: src/zkevm_specs/evm_circuit/execution/error_oog_sload_sstore.py (hunk 1 above)
Comment (KimiWu123) [reply src/zkevm_specs/evm_circuit/execution/error_oog_sload_sstore.py]: We don't need this `+1` anymore after merging `master` fixed in [73c2935](https://github.com/privacy-scaling-explorations/zkevm-specs/pull/473/commits/73c2935049b9cfa8ed082ea067514949c1e8b4cf)

**Review by rrtoledo — APPROVED** [review]
LGTM!

**Pull request merged**